azurelinuxagent/ga/update.py

```diff
@@ -525,6 +525,8 @@ def _ensure_readonly_files(self):
     def _ensure_cgroups_initialized(self):
         configurator = CGroupConfigurator.get_instance()
         configurator.initialize()
+        if configurator.enabled():
+            configurator.create_slices()
```

Comment:
the other methods in the configurator do "if configurator.enabled():" in their definition (i.e. we always invoke them and they are no-ops if cgroups are not enabled)